src/features/support/pages/SupportPage.tsx

```diff
@@ -1,7 +1,7 @@
 import { useContext, useEffect } from "react";
 import { ButtonBack } from "src/components/ButtonBack";
 import { ThemeContext } from "src/features/theme/ThemeContext";
-import { SearchInput } from "../components/SearchInput/SearchInput";
+import { SearchInput } from "../../../components/SearchInput/SearchInput";
```

Comment:
Please use relative path with baseUrl for global import and you missed the index path:
`"../../../components/SearchInput/SearchInput"` → `"src/components/SearchInput"`